db/hybrid_logical_vector.go

```diff
@@ -30,6 +31,85 @@ type Version struct {
 	Value uint64 `json:"version"`
 }
 
+// VersionsDeltas will be sorted by version, first entry will be fill version then after that will be calculated deltas
+type VersionsDeltas []Version
+
+func (vde VersionsDeltas) Len() int { return len(vde) }
+
+func (vde VersionsDeltas) Swap(i, j int) {
+	vde[i], vde[j] = vde[j], vde[i]
+}
+
+func (vde VersionsDeltas) Less(i, j int) bool {
+	if vde[i].Value == vde[j].Value {
+		return vde[i].SourceID < vde[j].SourceID
+	}
+	return vde[i].Value < vde[j].Value
+}
+
+// VersionDeltas calculate the deltas of input map
+func VersionDeltas(versions map[string]uint64) VersionsDeltas {
+	if versions == nil {
+		return nil
+	}
+
+	vdm := make(VersionsDeltas, 0, len(versions))
+	for src, vrs := range versions {
+		vdm = append(vdm, CreateVersion(src, vrs))
+	}
+
```

Comment:
Should we return early here when len(versions) == 1?  And do we have test coverage for that scenario?

Reply:
Giood idea, yes we have a scenario where we have single entry in PV in TestPvDeltaReadAndWrite